packages/unmock-core/src/service/interfaces.ts

```diff
@@ -22,6 +28,10 @@ export interface IServiceMapping {
   [serviceName: string]: IService;
 }
 
+export interface IEndpointToRegexMapping {
```

Comment:
Does this need an interface if it's only used internally by the matcher?

Reply:
I'd perhaps move it to `matcher.ts` and remove the `export`.

Reply:
Not really - it's just shorter than writing the mapping in several places and if we decide to change it for some reason, it takes effect in all relevant places.

I will move it to `matcher.ts` though.